firmware-support/memmap-generate/src/backends/c/mod.rs

```diff
@@ -217,7 +217,7 @@ fn po2_type(n: u64) -> u64 {
     }
 }
 
-fn tuple_name(
+pub fn tuple_name(
```

Comment:
this should be reverted too since the proper fix doesn't require this to be public